src/main/resources/application.properties

```diff
@@ -3,13 +3,13 @@ spring.jpa.hibernate.dll-auto=update
 spring.jpa.properties.hibernate.dialect=org.hibernate.dialect.MySQL8Dialect
 spring.datasource.url=jdbc:mysql://localhost:3306/hospital_appointment_management-db?createDatabaseIfNotExist=true&autoReconnect=true&useSSL=false&allowPublicKeyRetrieval=true
 spring.datasource.username=root
-spring.datasource.password=password
+spring.datasource.password=
 spring.jpa.properties.hibernate.validator.apply_to_dll=true
 spring.jpa.show-sql=true
 spring.jpa.database-platform=org.hibernate.dialect.MySQLDialect
 spring.jpa.hibernate.ddl-auto=update
 
 #security
-#spring.security.user.name=user
-#spring.security.user.password=password
+spring.security.user.name=user
```

Comment:
also comment these out again we dont use them, we define passwords and username in the securityCofig file